pyro/contrib/timeseries/gp.py

```diff
@@ -62,7 +62,7 @@ def _get_obs_dist(self):
 
     def _get_dist(self):
         """
-        Get the `GaussianHMM` distribution that corresponds to `obs_dim`-many independent Matern GPs.
+        Get the :class:`GaussianHMM` distribution that corresponds to ``obs_dim``-many independent Matern GPs.
```

Comment:
This needs to be fully qualified, and you can exclude the module name from rendered docs using tilde:
```
:class:`~pyro.distributions.GaussianHMM`
```